src/layouts/Authorizations/components/AddNewAuth/index.tsx

```diff
@@ -177,6 +178,18 @@ export const AddNewAuth: React.FC<IAddNewAuthProps> = ({ toggleModal }) => {
           </StyledExpandedTypeSelect>
         )}
       </StyledTypeSelectWrapper>
+      {selectedAuthType === 'RotatePrimaryKey' && (
+        <StyledWarningMessage>
+          Caution: This will remove the current primary key from your identity
+          and replace it with the target key
+        </StyledWarningMessage>
+      )}
+      {selectedAuthType === 'RotatePrimaryKeyToSecondary' && (
+        <StyledWarningMessage>
+          Note: This will make the current primary key a secondary key and
+          replace it with the target key
```

Comment:
Small note - I think the target has to be an existing secondary key that may be worth noting,

Reply:
As discussed, rotating to secondary key is allowed for either an existing secondary key or an unattached key